framework/encode/vulkan_capture_manager.cpp

```diff
@@ -2822,5 +2823,35 @@ void VulkanCaptureManager::PreProcess_vkBindImageMemory2(VkDevice
     }
 }
 
+void VulkanCaptureManager::PostProcess_vkCreateShaderModule(VkResult                        result,
+                                                            VkDevice                        device,
+                                                            const VkShaderModuleCreateInfo* pCreateInfo,
+                                                            const VkAllocationCallbacks*    pAllocator,
+                                                            VkShaderModule*                 pShaderModule)
+{
+    // spirv-parsing for buffer-references
+    gfxrecon::util::SpirVParsingUtil spirVParsingUtil;
+    printf("PostProcess_vkCreateShaderModule\n");
+
+    // TODO: adding this block compiles/links fine, but bricks the layer somehow!?
```

Comment:
this is where I'm currently stuck. 
commenting in this block will not issue any error/warning, but silently suppress the layer from loading/working.

Reply:
ok, avoiding usage of operator-new in SpirVParsingUtil solved the silent bail-out.